.environment-scripts/gaea/activate_conda_environment.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+CONDA_ENV=$1
+
+# We don't use module load python/3.9 since that pollutes the PATH
```

Comment:
This was bothering me this afternoon.  This was a solution I settled earlier, but I feel like there must be a way to avoid hardcoding this path to conda.  I’m going to inquire about at GFDL.  With the module load approach there’s an odd behavior in that activating a new environment doesn’t remove the base environment from the search path.

Reply:
Folks at Oak Ridge are currently looking into this.

Reply:
This has now been resolved on Gaea, so we can use a cleaner way of doing this.